src/test/java/com/epam/mjc/io/FileReaderTest.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.mjc.io;
```

Comment:
As I can see, this task for NIO, but package called `io`

Reply:
I agree

Reply:
Fixed